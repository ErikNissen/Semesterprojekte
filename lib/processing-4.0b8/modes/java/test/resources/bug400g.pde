version https://git-lfs.github.com/spec/v1
oid sha256:89f250b2222ed161c118b1481e3c927e1a59d6b514104b9ef1456e4282755a1f
size 143
