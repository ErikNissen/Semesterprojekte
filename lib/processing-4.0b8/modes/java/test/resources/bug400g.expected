version https://git-lfs.github.com/spec/v1
oid sha256:e25cc0ec1c921925c1135b820f3937230bad144fba5bf4e53473c3887bb10b6d
size 794
